### PR TITLE
PSMDB-745 Documented support of multiple LDAP servers

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: source/authentication.rst [hunk 1]
```diff
@@ -311,21 +311,20 @@ Kerberos Authentication
...
 As of version 3.6.18-5.0, |psmdb| supports |ldap-authorization|.
 
 This feature has been supported in |mongodb-e| since its version 3.4. 
   
-Note the following limitations of |ldap-authorization| in |psmdb|:
+.. note::
   
-- The `ldapTimeoutMS <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldaptimeoutms>`_ parameter is ignored.
-- The `--ldapServers <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldapservers>`_ option may only contain a single server (|mongodb-e| accepts a comma-separated list).
+   The `ldapTimeoutMS <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldaptimeoutms>`_ parameter is ignored.
```
Comment (igorsol) [inline source/authentication.rst]: ldapTimeoutMS is not ignored starting from version 3.6.18-5.0. Its value is used as timeout values when executing LDAP queries.

**Review by igorsol — CHANGES_REQUESTED** [review]

**Review by akira-kurogane — APPROVED** [review]
LGTM

**Review by igorsol — APPROVED** [review]

File: source/authentication.rst [hunk 2]
```diff
@@ -351,12 +346,28 @@ Alternatively, edit the configuration file:
    setParameter:
      ldapConnectionPoolSizePerHost: 5
 
+.. rubric:: Support for multiple LDAP servers
+
+As of version 3.6.21-11.0, you can specify multiple LDAP servers for failover. |PSMDB| sends authentication requests to the first server defined in the list. When this server is down or unavailable, it sends requests to the next server  and so on. Note that |PSMDB| keeps sending requests to this server even after the unavailable server recovers.
+
+Specify the LDAP servers as a comma-separated list in the format ``<host>:<port>`` for the `--ldapServers <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldapservers>`_ option.
```
Comment (sandraromanchenko) [inline source/authentication.rst]: Wrong link
https://docs.mongodb.com/manual/reference/program/mongod/index.html#cmdoption-mongod-ldapservers

File: source/authentication.rst [hunk 3]
```diff
@@ -351,12 +346,28 @@ Alternatively, edit the configuration file:
...
+Specify the LDAP servers as a comma-separated list in the format ``<host>:<port>`` for the `--ldapServers <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldapservers>`_ option. 
+
+.. code-block:: yaml
+
+   security:
+     authorization: "enabled"
+     ldap:
+       servers: "ldap1.example.net", "ldap2.example.net"
+
+You can set the option value either at the server startup or dynamically at runtime using the :ref:`setParameter <setParameter>`.
```
Comment (sandraromanchenko) [inline source/authentication.rst]: Option should be set/defined at the startup, but can be changed at runtime

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: source/authentication.rst [hunk 4]
```diff
@@ -351,12 +346,28 @@ Alternatively, edit the configuration file:
...
+As of version 3.6.21-11.0, you can specify multiple LDAP servers for failover. |PSMDB| sends authentication requests to the first server defined in the list. When this server is down or unavailable, it sends requests to the next server  and so on. Note that |PSMDB| keeps sending requests to this server even after the unavailable server recovers.
+
+Specify the LDAP servers as a comma-separated list in the format ``<host>:<port>`` for the `--ldapServers <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldapservers>`_ option. 
+
+.. code-block:: yaml
+
+   security:
+     authorization: "enabled"
+     ldap:
+       servers: "ldap1.example.net", "ldap2.example.net"
```
Comment (nastena1606) [inline source/authentication.rst]: Is it a correct definition? Or should values be included in a single quotation marks?

Comment (sandraromanchenko) [reply source/authentication.rst]: Nope, the correct one is either 
servers: "ldap1.example.net,ldap2.example.net" 
OR
servers: ldap1.example.net,ldap2.example.net

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**